applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -113,10 +114,12 @@ LOG_MODULE_REGISTER(audio_datapath, CONFIG_LOG_AUDIO_DATAPATH_LEVEL);
 #define DRIFT_ERR_THRESH_LOCK 16
 #define DRIFT_ERR_THRESH_UNLOCK 32
 
-// TODO: Re-enable
-#define PRES_COMP_ENABLE false
-/* Presentation delay in microseconds */
-#define PRES_DLY_US 10000
+#define PRES_COMP_ENABLE true
+
+/* Just in time defines */
+/* 3000 us to allow BLE transmission and (host -> HCI -> controller) */
+#define JUST_IN_TIME_US (CONFIG_AUDIO_FRAME_DURATION_US - 3000)
+#define JIT_THRESHOLD_US 1500
```

Comment:
I suggest changing the name to JUST_IN_TIME_THRESHOLD_US to keep consistency, despite the name being longer